plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/SerdeConstants.java

```diff
@@ -28,5 +28,9 @@ public final class SerdeConstants
     public static final String LIST_COLUMN_TYPES = "columns.types";
     public static final String LIST_COLUMN_COMMENTS = "columns.comments";
 
+    public static final String ION_ENCODING = "ion.encoding";
+    public static final String TEXT_ENCODING = "Text";
+    public static final String BINARY_ENCODING = "Binary";
```

Comment:
We should probably keep all the Ion related constants in `IonFileWriterFactory.java` or `IonFileWriter.java` where it is used. This file seems to contain more common constants for hive serde.

Reply:
We're also gonna end up with a whole bunch of these.